### PR TITLE
Add accent to piraíba and tucunaré

### PR DESCRIPTION
**Pull request by Faalagorn**
#### Summary
None

#### Purpose of change
Use Unicode letter when appropriate.

#### Describe the solution
Change `i` to `í` in piraíba based on https://en.wikipedia.org/wiki/Brachyplatystoma_filamentosum and `e` to `é` to tucunaré https://en.wikipedia.org/wiki/Peacock_bass.

#### Describe alternatives you've considered
None.

#### Testing
Simple text change, can be seen in debug with TropiCataclysm enabled.

#### Additional context
I think it's the only instance of those names?

### CODE REVIEW

File: data/mods/TropiCataclysm/monsters/tropical_fish.json [hunk 1]
```diff
@@ -153,7 +153,7 @@
     "id": "mon_fish_tucunare",
     "type": "MONSTER",
     "copy-from": "mon_fish_large",
-    "name": { "str": "tucunare" },
+    "name": { "str": "tucunaré" },
```
Comment (BrettDong) [inline data/mods/TropiCataclysm/monsters/tropical_fish.json:156]: You need to manually specify plural form for this one.

Comment (Faalagorn) [reply data/mods/TropiCataclysm/monsters/tropical_fish.json:156]: Hm, so it doesn't add plurals if it doesn't end with the regular character then?

Comment (BrettDong) [reply data/mods/TropiCataclysm/monsters/tropical_fish.json:156]: The failed test says "Cannot autogenerate plural form.  Please specify the plural form explicitly."

**Comment by OnTheDouble** [comment]
Non-standard letters lead to quality of life issues like these https://github.com/CleverRaven/Cataclysm-DDA/issues/56938

@kevingranade please reconsider your policy on this, it makes UX worse.

**Comment by Faalagorn** [comment]
That's a good point; which is also hardened as sometimes the keybindings collide with the method to (easily) input the character. I guess the best way would be to be able to enter the unaccented letter which would then show the unaccented versions, kinda similar to how e.g. Firefox or other modern software are allowing to. I guess it's best to made a separate issue about it @OnTheDouble?

**Comment by OnTheDouble** [comment]
I am fine with accents if you guys make it so that when I type _paraiba_ I find _piraíba_. I respect foreign rules but it's best that our search works much like Google where it forgives the player for not following some of the spelling.

Is that something that's possible to do?

**Comment by Faalagorn** [comment]
I sincerely hope it's possible, though I'm probably not the one with the technical feat to implement it. I guess it's best to move the discussion to #56938 as closed PRs are less visible!

**Pull request merged**